src/odemis/model/_vattributes.py

```diff
@@ -1123,6 +1123,33 @@ def _check(self, value):
                     (axis, v, self._range[0], self._range[1]))
 
 
+class ListContinuous(ListVA, Continuous):
+    def __init__(self, value, range, unit="", cls=None, **kwargs):
+        """
+        range (2 x list): minimum and maximum size for each dimension
+        cls (class or list of classes): classes allowed for each element of the tuple
+          default to the same class as the first element
+        """
+        if not isinstance(value, list):
+            raise ValueError("value (%s) is not a list" % (value,))
+        self._cls = cls or value[0].__class__
+        Continuous.__init__(self, range)
+        ListVA.__init__(self, value, unit=unit, **kwargs)
+
+    def _set_value(self, value, **kwargs):
+        # force list
+        value = list(value)
```

Comment:
No need for this, as `ListVA._set_value()` does this too.

Reply:
It made an issue with me while terminating the multiplex tests. The dependencies `terminate()` sets power with range[0] and it stayed tuple.